lib/get-macro-options/index.js

```diff
@@ -101,7 +101,8 @@ function getAdditionalComponentOptions(options, parent) {
     .map((option) => {
       const output = []
 
-      if (option.isComponent && ['hint', 'label'].includes(option.slug)) {
+      const hasComponentPage = !['hint', 'label'].includes(option.slug)
+      if (option.isComponent && !option.params && !hasComponentPage) {
```

Comment:
I've added checks for `option.params` as a safeguard

For some nested `isComponent` options we may add `option.params` to show the subset of available options in a table, rather than link out to the component page where all options are shown

See [related](https://trello.com/c/smPF3ePN/18-nested-components-components-used-by-other-components-should-not-have-their-params-listed-in-the-top-level-component-instead-sho) [cards](https://trello.com/c/RxuFrKQS/19-radio-and-checkbox-item-nested-label-components-list-options-that-cant-be-used) on Trello